examples/language/lib/functions/getters_setters.dart

```diff
@@ -0,0 +1,42 @@
+/// Defines a private variable secret with `_`.
+String _secret = 'Hello';
+
+/// The public top-level getter.
+/// It provides read access to the [_secret].
```

Comment:
Since this is more intended as context for understanding the code snippet rather than how [doc comments](https://dart.dev/language/comments#documentation-comments) should be written, consider converting this to normal [inline comments](https://dart.dev/language/comments#single-line-comments).

Perhaps similar to this:

```dart
// A public top-level getter that
// provides read access to [_secret].
```